docs/index.md

```diff
@@ -12,12 +12,28 @@ This pipeline is part of [BioWDL](https://biowdl.github.io/)
 developed by [the SASC team](http://sasc.lumc.nl/).
 
 ## Usage
+To run the pipeline, one should first clone the git repository into their excecution folder. Use something like:
+```bash
+git clone --recursive  https://github.com/biowdl/RNA-seq.git
+```
+This will create a copy of all `.wdl` scripts and other files necessary for the pipeline run (eg. `dockerTags.yml`)
+
 This pipeline can be run using
 [Cromwell](http://cromwell.readthedocs.io/en/stable/):
 ```bash
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
+If you want to run the pipeline on Shark, you should first load the cromwell module with something like:
```

Comment:
I'm wondering whether this should be here, or in [its own separate documentation](https://git.lumc.nl/sasc/cromwell-on-shark/wikis/Cromwell-on-Shark)
@hailiangmei @rhpvorderman Thoughts?

Reply:
I did not know about this separate docs. I see it is a wiki available only within LUMC. If we can add a link to the wiki, the docs are obviously unnecessary.